models/staging/stripe/stg_payments.sql

```diff
@@ -0,0 +1,3 @@
+select orderid as order_id, sum(amount) as amount from 
+raw.stripe.payment where status = 'success'

```

Comment:
same as above -- this is order sums -- the staging model for payments should just be pulling through everything from payments at the `payment_id` grain -- aggregation then can happen downstream